mux/src/tmux_pty.rs

```diff
@@ -0,0 +1,118 @@
+use crate::{
+    tmux::{RefTmuxRemotePane, TmuxCmdQueue, TmuxDomainState},
+    tmux_commands::SendKeys,
+};
+use flume;
```

Comment:
using just the crate name doesn't really "do" anything, so you should be able to remove it!

```suggestion
```